app/recipe/serializers.py

```diff
@@ -0,0 +1,65 @@
+from rest_framework import serializers
+from core.models import Recipe, Ingredient
+
+
+class IngredientSerializer(serializers.ModelSerializer):
+    """
+    Serializer for Ingredient objects
+    """
+    print('IngredientSerializer')
```

Comment:
Avoid adding prints in the middle of code which can litter the console output on startup.